### PR TITLE
Add config.yml for ISSUE_TEMPLATE so that Doc, Security, Support, and WinPS issues go to URLs

### PR DESCRIPTION
**Pull request by SteveL-MSFT**
# PR Summary

Create `config.yml` for ISSUE_TEMPLATE folder so that `New Issue` will go to relevant links instead of allowing user to create such an issue which is not appropriate for this repo.

You can see what this looks like by pressing `New Issue` here: https://github.com/stevel-msft/test/issues

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [x] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary. This may include:
        - Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode (which runs in a different PS Host).
        - Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
        - Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
        - Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    about: Windows PowerShell issues or suggestions.
+  - name: Support
+    url: https://github.com/PowerShell/PowerShell/blob/master/.github/SUPPORT.md
+    about: PowerShell Support Questions/Help
+  - name: Documentation Issue
+    url: https://github.com/MicrosoftDocs/PowerShell-Docs
+    about: Please open issues on documentation for PowerShell here.
+  - name: Security Issue
+    url: https://github.com/PowerShell/PowerShell/blob/master/.github/SECURITY.md
+    about: Reporting security issues for PowerShell.
```
Comment (iSazonov) [inline .github/ISSUE_TEMPLATE/config.yml]: We could add
https://github.com/PowerShell/PowerShellModuleCoverage
https://github.com/PowerShell/PSReadLine
https://github.com/Azure/azure-powershell

File: .github/ISSUE_TEMPLATE/Bug_Report.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ assignees: ''
 For Windows PowerShell 5.1 issues, suggestions, or feature requests please use the following link instead:
 Windows PowerShell [UserVoice](https://windowsserver.uservoice.com/forums/301869-powershell)
 
-This repository is **ONLY** for PowerShell Core 6 issues.
+This repository is **ONLY** for PowerShell Core 6 and PowerShell 7 issues.
```
Comment (mklement0) [inline .github/ISSUE_TEMPLATE/Bug_Report.md]: Why not just say "PowerShell Core" (version 6 or higher) issues", or, with a view to possibly dropping the "Core" qualifier soonish, "for issues relating to PowerShell version 6 and above"

**Comment by mklement0** [comment]
* Re the "Support" link: I think it should link to a page that contains the same information that is in the current template, whose gist is: don't post general support questions HERE; instead, go to Slack, ...

  * If you link to https://github.com/PowerShell/PowerShell/blob/master/.github/SUPPORT.md, as you currently propose, you may send users back to the place they came from.

* Re "Windows PowerShell" link: I think an intermediate page that explains the two editions and their maintenance status would be helpful.

* Re "Bug report": I assume what you're currently linking to is just a placeholder, and the current template will be retained.

File: .github/ISSUE_TEMPLATE/Bug_Report.md (hunk 2 above)
Comment (SteveL-MSFT) [reply .github/ISSUE_TEMPLATE/Bug_Report.md]: Officially, we're dropping `Core` from the name hence forth.  Current thinking is to stay on 7.x so I don't expect a PS8 for some time if ever

Comment (mklement0) [reply .github/ISSUE_TEMPLATE/Bug_Report.md]: Thanks for clarifying, @SteveL-MSFT - how does the "vNext" you were referring to [here](https://github.com/PowerShell/PowerShell/issues/1995#issuecomment-552533786) fit into the picture? Were just referring to 7.1?

Comment (TravisEz13) [reply .github/ISSUE_TEMPLATE/Bug_Report.md]:
```suggestion
This repository is **ONLY** for PowerShell Core 6 and PowerShell 7+ issues.
```

File: .github/ISSUE_TEMPLATE/Security_Issue_Report.md [hunk 3]
```diff
@@ -1,20 +0,0 @@
----
-name: Security Issue
```
Comment (TravisEz13) [inline .github/ISSUE_TEMPLATE/Security_Issue_Report.md]: Please don't delete this.  It makes sure that I get assigned security issues right away

File: .github/ISSUE_TEMPLATE/Documentation_Issue.md [hunk 4]
```diff
@@ -1,13 +0,0 @@
----
-name: Documentation Issue 📚
-about: File issues regarding documentation within the [PowerShell-Docs](https://github.com/MicrosoftDocs/PowerShell-Docs) repository
-title: "Documentation Issue"
-labels: Issue-Question,Area-Documentation
```
Comment (TravisEz13) [inline .github/ISSUE_TEMPLATE/Documentation_Issue.md]: This template help for tagging.

File: .github/ISSUE_TEMPLATE/Security_Issue_Report.md (hunk 3 above)
Comment (TravisEz13) [reply .github/ISSUE_TEMPLATE/Security_Issue_Report.md]: Also, they are allowed to file issues, if MSRC says to, and I'd prefer they have a template to do that.

Comment (TravisEz13) [reply .github/ISSUE_TEMPLATE/Security_Issue_Report.md]: Please answer my comments

File: .github/ISSUE_TEMPLATE/Documentation_Issue.md (hunk 4 above)
Comment (TravisEz13) [reply .github/ISSUE_TEMPLATE/Documentation_Issue.md]: Not blocking...

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (SteveL-MSFT) [reply .github/ISSUE_TEMPLATE/config.yml]: I'm going to leave those off for now as I don't see many issues coming in that necessitate adding them.  Don't want to have a huge menu that makes it more confusing.

File: .github/ISSUE_TEMPLATE/Security_Issue_Report.md (hunk 3 above)
Comment (SteveL-MSFT) [reply .github/ISSUE_TEMPLATE/Security_Issue_Report.md]: Will add back

**Review by TravisEz13 — APPROVED** [review]

**Pull request merged**